### PR TITLE
Add Setting to skip confirmation of marking feeds read

### PR DESCRIPTION
**Pull request by tarishij17**

### CODE REVIEW

File: doc/configcommands.dsv [hunk 1]
```diff
@@ -12,6 +12,7 @@ browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use whe
 cache-file||<path>||"~/.newsboat/cache.db" or "~/.local/share/cache.db" (see "Files" section)||This configuration option sets the cache file. This is especially useful if the filesystem of your home directory doesn't support proper locking (e.g. NFS).||cache-file "/tmp/testcache.db"
 cleanup-on-quit||[yes/no]||yes||If set to `yes`, then the cache gets locked and superfluous feeds and items are removed, such as feeds that can't be found in the urls configuration file anymore.||cleanup-on-quit no
 color||<element> <fgcolor> <bgcolor> [<attribute> ...]||n/a||Set the foreground color, background color and optional attributes for a certain element.||color background white black
+confirm-markAllFeedsRead||[yes/no]||yes||If set to `yes`, then newsboat will ask for confirmation whether the user wants to mark all feeds as read.||confirm-markAllFeedsRead no
```
Comment (Minoru) [inline doc/configcommands.dsv:15]: Please replace camelCase to be in line with other settings: `confirm-mark-all-feeds-read`.

File: src/feedlistformaction.cpp [hunk 2]
```diff
@@ -406,7 +406,9 @@ bool FeedListFormAction::process_operation(Operation op,
 	}
 	break;
 	case OP_MARKALLFEEDSREAD:
-		if (v->confirm(_("Do you really want to mark all feeds as read (y:Yes n:No)? "),
+		if (!get_cfg()->get_configvalue_as_bool(
```
Comment (Minoru) [inline src/feedlistformaction.cpp:409]: This fails to compile, can you please fix that? Here's the build log from the CI: https://travis-ci.com/github/newsboat/newsboat/jobs/394842896#L611

**Review by Minoru — CHANGES_REQUESTED** [review]
This looks pretty good!

We're using two CI providers, and one of them apparently refused to build the code because you just registered with GitHub :) I'll go ask the support what's up with that. I may have to delay merging this because of that — sorry.

**Comment by Minoru** [comment]
Also, out of curiosity: did you find this repo and issue through Hacktoberfest? I'm trying it for the first time this year, and would like to know if it indeed helps attract more contributors.

**Comment by tarishij17** [comment]
> Also, out of curiosity: did you find this repo and issue through Hacktoberfest? I'm trying it for the first time this year, and would like to know if it indeed helps attract more contributors.

yeah, it showed up in the search of Hacktoberfest labelled issues. I guess by adding Hacktoberfest label with the issue would be sufficient for other contributors to look into it.

**Comment by tarishij17** [comment]
I'll make the required changes later today

**Comment by tarishij17** [comment]
@Minoru plz review the PR for requested changes

**Comment by tarishij17** [comment]
@Minoru if the changes looks good , can you please label the PR as "hacktoberfest-accepted" so that it counts toward hactoberfest.

File: src/feedlistformaction.cpp [hunk 3]
```diff
@@ -406,7 +406,9 @@ bool FeedListFormAction::process_operation(Operation op,
 	}
 	break;
 	case OP_MARKALLFEEDSREAD:
-		if (v->confirm(_("Do you really want to mark all feeds as read (y:Yes n:No)? "),
+		if (!cfg->get_configvalue_as_bool(
```
Comment (Minoru) [inline src/feedlistformaction.cpp:409]: Please fix [the formatting issue](https://cirrus-ci.com/task/6433402410762240?command=main#L255) here.

**Review by Minoru — APPROVED** [review]
Looks (and works) great! Please fix the little formatting issue spotted by CI, and I'll finally merge this!

**Comment by Minoru** [comment]
> @Minoru if the changes looks good , can you please label the PR as "hacktoberfest-accepted" so that it counts toward hactoberfest.

I expect this PR will get merged in the next couple days anyway, so I'll save myself the trouble of keeping another label around. ;)

File: src/feedlistformaction.cpp (hunk 3 above)
Comment (tarishij17) [reply src/feedlistformaction.cpp:409]: just to confirm- so is this the indentation of the pointed line which needs to be fixed?

**Comment by tarishij17** [comment]
done formatting

**Comment by Minoru** [comment]
Excellent! Thank you for the work, @tarishij17! If you enjoyed working on Newsboat, perhaps you'd take a look at other issues? [I labelled a few](https://github.com/newsboat/newsboat/labels/Hacktoberfest) that might be a good start for someone new to the project, but feel free to take any you like — as far as I understand, it doesn't matter for Hacktoberfest what issues you're working on, or if there is an issue at all.

(Travis jobs already finished, but the results never made it to GitHub. That happens. I'm merging this anyway.)

**Comment by tarishij17** [comment]
Thanks @Minoru .It was nice working with you!

**Pull request merged**